x/trigger/module/module.go

```diff
@@ -0,0 +1,198 @@
+func (AppModuleBasic) RegisterRESTRoutes(_ sdkclient.Context, _ *mux.Router) {}
+
+// RegisterGRPCGatewayRoutes registers the gRPC Gateway routes for the trigger module.
+func (a AppModuleBasic) RegisterGRPCGatewayRoutes(clientCtx sdkclient.Context, mux *runtime.ServeMux) {
+	if err := types.RegisterQueryHandlerClient(context.Background(), mux, types.NewQueryClient(clientCtx)); err != nil {
+		panic(err)
+	}
+}
+
+// GetQueryCmd returns the cli query commands for the trigger module
+func (AppModuleBasic) GetQueryCmd() *cobra.Command {
+	return cli.GetQueryCmd()
+}
+
+// GetTxCmd returns the transaction commands for the trigger module
+func (AppModuleBasic) GetTxCmd() *cobra.Command {
+	return cli.NewTxCmd()
+}
+
+// AppModule implements the sdk.AppModule interface
+type AppModule struct {
+	AppModuleBasic
+	keeper        keeper.Keeper
+	accountKeeper authkeeper.AccountKeeper
+	bankKeeper    bankkeeper.Keeper
+}
+
+// NewAppModule creates a new AppModule object
+func NewAppModule(cdc codec.Codec, keeper keeper.Keeper, accountKeeper authkeeper.AccountKeeper, bankKeeper bankkeeper.Keeper) AppModule {
+	return AppModule{
+		AppModuleBasic: AppModuleBasic{cdc: cdc},
+		keeper:         keeper,
+		accountKeeper:  accountKeeper,
+		bankKeeper:     bankKeeper,
+	}
+}
+
+// GenerateGenesisState creates a randomized GenState of the trigger module.
+func (am AppModule) GenerateGenesisState(simState *module.SimulationState) {
+	simulation.RandomizedGenState(simState)
+}
+
+// ProposalContents returns content functions used to simulate governance proposals.
+func (am AppModule) ProposalContents(_ module.SimulationState) []simtypes.WeightedProposalContent {
+	// currently no gov proposals exist
+	return nil
+}
+
+// RandomizedParams returns randomized module parameters for param change proposals.
+func (am AppModule) RandomizedParams(_ *rand.Rand) []simtypes.ParamChange {
+	// currently no module params exist
+	return nil
+}
+
+// RegisterStoreDecoder registers a func to decode each module's defined types from their corresponding store key
+func (am AppModule) RegisterStoreDecoder(sdr sdk.StoreDecoderRegistry) {
+	sdr[keeper.StoreKey] = simulation.NewDecodeStore(am.cdc)
+}
+
+// WeightedOperations returns simulation operations (i.e msgs) with their respective weight
+func (am AppModule) WeightedOperations(simState module.SimulationState) []simtypes.WeightedOperation {
+	return simulation.WeightedOperations(
+		simState.AppParams, simState.Cdc, am.keeper, am.accountKeeper, am.bankKeeper,
+	)
+}
+
+// Name returns the trigger module's name.
+func (AppModule) Name() string {
+	return types.ModuleName
+}
+
+// RegisterInvariants does nothing, there are no invariants to enforce
+func (AppModule) RegisterInvariants(_ sdk.InvariantRegistry) {}
+
+// Deprecated: Route returns the message routing key for the trigger module.
+func (am AppModule) Route() sdk.Route {
+	return sdk.NewRoute(types.RouterKey, triggerModule.NewHandler(am.keeper))
+}
```

Comment:
This is deprecated. You can just return `return sdk.Route{}` here. And that means you can get rid of the handler too.